### DISCUSSION TITLE
Default sales order items plugin

### DISCUSSION THREAD

**Discussion by lippoliv** [inventree/InvenTree#5635] [Show and tell]
Hi there,
on my "road to InvenTree" I implement several plugins, mostly internal. But here's one which may be a rare usecase as well but at least good to share for who's interested: https://github.com/lippoliv/inventree-default-salesorder-items

With this plugin you can specify a list of part ids in the settings which then will be added to every created sales order automatically.

**Comment by matmair** [comment]
@lippoliv if you want the plugin can be added to the [official plugin registry](https://inventree.org/plugins) with a PR here: https://github.com/inventree/inventree-website

**Reply by lippoliv** [reply]
Nice, thanks. Here we go https://github.com/inventree/inventree-website/pull/168
